Vagrantfile

```diff
@@ -174,28 +170,23 @@ Vagrant.configure(2) do |config|
   config.hostmanager.ignore_private_ip = false
 
   # configure vagrant-vbguest plugin
-  if Vagrant.has_plugin?('vagrant-vbguest')
```

Comment:
prefer this not be inline. gonna have to add another plugin check soon, want them to be easy to spot.